TestFoundation/TestNSHTTPCookieStorage.swift

```diff
@@ -227,18 +227,22 @@ class TestNSHTTPCookieStorage: XCTestCase {
         storage.setCookie(testCookie)
         XCTAssertEqual(storage.cookies!.count, 3)
         var destPath: String
+        //let bundleName = Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as! String
+        //TODO: Above api returns nil on Linux when this code is introduced. Hence, using the below workaround.
+        //Needs replacement once fixed.
+        let bundlePath = Bundle.main.bundlePath
+        var bundlePathComponents = bundlePath.components(separatedBy: "/")
+        let bundleName: String = "/" + bundlePathComponents[bundlePathComponents.count - 1]
```

Comment:
This won't do the right thing for executables. The last component will just be the parent directory.

The `: String` is not necessary as it will be inferred.